src/testRunner/unittests/tsbuildWatch/projectsBuilding.ts

```diff
@@ -11,29 +11,31 @@ namespace ts.tscWatch {
             return { path: `./pkg${index}` };
         }
         function pkgFiles(index: number): File[] {
+            const tsconfig = {
+                compilerOptions: { composite: true },

```

Comment:
Were these changes necessary to make the code compile?

Reply:
Yes.

Reply:
is it because of contextual typing? or maybe object literal freshness?